Makefile

```diff
@@ -204,12 +204,12 @@ coverage:
 COVER_REPORT_OPTS ?= -select_re '^(lib|script|t)/'
 
 .PHONY: coverage-codecov
-coverage-codecov: coverage
-	cover $(COVER_REPORT_OPTS) -report codecov
+coverage-codecov:
+	cover $(COVER_REPORT_OPTS) -report codecovbash
 
 .PHONY: coverage-html
-coverage-html: coverage
-	cover $(COVER_REPORT_OPTS) -report html_basic
+coverage-html:
```

Comment:
This is changing the previous notion of `coverage-html` implying that the coverage is being collected. What is the specific reason you needed to change this?

Reply:
Because I wasted some of my time investigating with incorrectly replicating the steps done in CI. I'd like to merge the steps. At least this is a step in that direction.

Reply:
Sorry, I don't understand. The line `cover $(COVER_REPORT_OPTS) -report html_minimal` only creates the final report but does not conduct any tests to initially collect the coverage data. Where do you "merge steps"?

Reply:
I meant making sure CI uses the same makefile targets humans use. The HTML didn't even work for me when I tried it locally, so I don't think it was used at all.